.github/scripts/tests/test-toolchain.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I think this file would better fit to `.github/scripts/toolchain` folder being named `execute-binutils-tests.sh` there.